pkgs/development/tools/java/java-language-server/default.nix

```diff
@@ -24,7 +24,7 @@ javaPackages.mavenfod rec {
 
   mvnFetchExtraArgs.dontConfigure = true;
   mvnParameters = "-DskipTests";
-  mvnHash = "sha256-2MVlF3QIWiDvUlnMH4RLi2Od57aoh8zK/OmHqztOnZ4=";
+  mvnHash = "sha256-bzYBSrCS9Kp+qnVO60h915Or1VWabphwLEu6lcBULuc=";
```

Comment:
(note to self) this project seems kind of dead. there are really trivial PRs that haven't been merged in months :(

I threw in the enforce PR just to kind of see what will happen: georgewfraser/java-language-server#259

Reply:
nevermind, this was merged :)